src/pages/experiments/[experimentId]/data-exploration/index.jsx

```diff
@@ -60,7 +60,7 @@ const ExplorationViewPage = ({
     Heatmap: {
       toolbarControls: (
         <>
-          <HeatmapSettings componentType={COMPONENT_TYPE} key='heatmap-settings' />
+          <HeatmapSettings componentType={COMPONENT_UUID} key='heatmap-settings' />
```

Comment:
I know I asked you to make everything the same type and this is why you changed it to component_uuid, but I didn't realise that now this is going to be confusing here: `componentType={COMPONENT_UUID}`. Sounds like it was maybe better to use component type for data exploration and component uuid for plots and tables. Can you change that back? Sorry!